app/assets/javascripts/discourse/app/widgets/post-menu.js

```diff
@@ -565,6 +565,15 @@ export default createWidget("post-menu", {
       }
     }
 
+    const extraPostControls = applyDecorators(
+      this,
+      "extra-post-controls",
```

Comment:
can we use: `after-post-controls` instead ?

Reply:
As we want it inside the post-controls nav, so I think it's okay if we keep it `extra-post-controls`